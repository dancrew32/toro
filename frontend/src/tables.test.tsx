import * as React from "react";
import * as TestRenderer from "react-test-renderer";

import Tables from "./tables";

test("<Tables />", () => {
  const tree = TestRenderer.create(<Tables />).toJSON();
  expect(tree).toMatchSnapshot();
});
