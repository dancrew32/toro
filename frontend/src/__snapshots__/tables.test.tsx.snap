// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tables /> 1`] = `
<div
  className="container"
>
  <nav>
    <a
      className="active"
      href="/"
      onClick={[Function]}
    >
      Toro
    </a>
    <a
      className=""
      href="/tables"
      onClick={[Function]}
    >
      Tables
    </a>
  </nav>
  <aside>
    <div
      className="inner"
    >
      <div>
        <h3>
          Tables
        </h3>
        <div
          className="search"
        >
          <input
            autoFocus={true}
            onChange={[Function]}
            placeholder="Search..."
            type="text"
            value=""
          />
        </div>
      </div>
    </div>
  </aside>
  <main>
    <div
      className="inner"
    >
      <div>
        Please select a schema & table to view its metrics.
      </div>
    </div>
  </main>
</div>
`;
